### PR TITLE
WIP - import a dirty script for uploading to S3.

### PR DESCRIPTION
**Pull request by rhettg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,12 +285,28 @@ For example:
...
+
+    oxstore prune --log-path=/var/log/blueox --keep-days=7
+
+Retrieve your log data over a time span
+
+    oxstore cat --log-path=/var/log/blueox --start=20120313 --end=20120315 request
+
+Store your zipped logs in an S3 bucket:
+
+    oxstore archive --log-path=/var/log/blueox --bucket=com.example.ox
```
Comment (bordicon) [inline README.md]: issues with overwriting?

File: README.md [hunk 2]
```diff
@@ -278,12 +285,28 @@ For example:
...
-return all events with a type that begins with 'request'
+    oxstore zip --log-path=/var/log/blueox
+
+If you don't need to keep everything:
+
+    oxstore prune --log-path=/var/log/blueox --keep-days=7
+
+Retrieve your log data over a time span
+
+    oxstore cat --log-path=/var/log/blueox --start=20120313 --end=20120315 request
```
Comment (bordicon) [inline README.md]: thanks for adding these examples

File: README.md (hunk 1 above)
Comment (bordicon) [inline README.md]: right does not overwrite if key exists

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 testify
 pep8
 pyflakes
-tornado==3.2
+tornado==3.2
```
Comment (bordicon) [inline requirements.txt]: what is tornado for in this PR?

File: bin/oxstore [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+        'log_type', action='store', nargs=1, help="log type")
+    parser_dnld.add_argument(
+        '--start', '-s',
+        action='store', required=True, help="start date (YYYYMMDD)")
+    parser_dnld.add_argument(
+        '--end', '-e', action='store', required=False, help="end date")
+
+    args = parser.parse_args()
+
+    if boto is None:
```
Comment (bordicon) [inline bin/oxstore]: why are we testing for this specific case?

File: README.md (hunk 2 above)
Comment (rhettg) [inline README.md]: they don't actually all work yet :expressionless:

File: README.md (hunk 1 above)
Comment (rhettg) [inline README.md]: We should probably enhance it to update if it's larger or something.

File: bin/oxstore (hunk 4 above)
Comment (rhettg) [inline bin/oxstore]: I want to make this script work if you don't have boto installed. Like you can use all the none-s3 features. But that's obviously not yet how it's structured.

You'll notice I only added boto to requirements.txt, not to the setup.py dependencies.

File: requirements.txt (hunk 3 above)
Comment (rhettg) [inline requirements.txt]: i think it's just a weird diff.

**Pull request closed without merging**